.github/workflows/events-recurrent.yml

```diff
@@ -13,9 +13,25 @@ jobs:
     name: Label PR conflits
     runs-on: ubuntu-20.04
     steps:
-      - uses: mschilde/auto-label-merge-conflicts@v2
+      - uses: mschilde/auto-label-merge-conflicts@v2.0
         with:
           CONFLICT_LABEL_NAME: "has conflicts"
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           MAX_RETRIES: 3
           WAIT_MS: 5000
+
+  # autoupdate is a GitHub Action that auto-updates pull requests branches whenever changes land on their destination branch.
+  # see: https://github.com/marketplace/actions/auto-update
+  pr-auto-update:
+    name: Auto-update PR
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: docker://chinthakagodawita/autoupdate-action:v1
+        env:
+          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+          DRY_RUN: "false"
+          PR_FILTER: "labelled"
```

Comment:
Shouldn't you filter `has conflicts` ?

Reply:
This does not seem to be available for this. In the mentioned action it states that this is coming soon...

Reply:
good point, adding it here #5480